### DISCUSSION TITLE
how to config yazi to disable image previewer?

### DISCUSSION THREAD

**Discussion by WingDust** [sxyazi/yazi#2001] [1. Q&A]
### What system are you running Yazi on?

Windows

### What terminal are you running Yazi in?

0.3.3

### `yazi --debug` output

```Shell
not find in issue
```

### Describe the question

how to config yazi to disable image previewer?

### Checklist

- [X] I have searched the existing discussions/issues

**Comment by sxyazi** [comment]
Use the `noop` built-in previewer to disable it, see https://yazi-rs.github.io/docs/configuration/yazi/#plugin.previewers and https://yazi-rs.github.io/docs/tips#folder-previewer

**Reply by WingDust** [reply]
```
[plugin]
  prepend_previewers = [
    { mime = "image/*", run = "noop" },
    { mime = "image/{avif,hei?,jxl}", run = "noop" },
  ]
```
this is right ?

**Reply by sxyazi** [reply]
Yeah that's right. If you also want to disable preloaders use prepend_preloaders
